jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererCodeCoverageTracker.kt

```diff
@@ -71,6 +83,26 @@ abstract class CodeWhispererCodeCoverageTracker(
         addAndGetTotalTokens(event.newLength - event.oldLength)
     }
 
+    internal fun extractRangeMarkerString(rangeMarker: RangeMarker?): String? = runReadAction {
+        rangeMarker?.range?.let { myRange -> rangeMarker.document.getText(myRange) }
+    }
+
+    internal fun getAcceptedTokensDelta(originalRecommendation: String, modifiedRecommendation: String): Int {
+        val editDistance = getEditDistance(modifiedRecommendation, originalRecommendation).toInt()
```

Comment:
can you add a comment here why using editDistance makes the calculation correct? Comments would help the algorithm used here.

Reply:
yea I will add some comment or example here for demonstration, but the case is it is still not 100% capturing the users' modification as same as UserModification, but it's the best effort I can think of for now and IMO it can capture most of the cases and provide a good approximate on users' modification.